### ISSUE TITLE
normalize notes for codeflags

### ISSUE THREAD

**Issue by amilan17** [wmo-im/GRIB2#98]
# Branch

[https://github.com/wmo-im/GRIB2/tree/codeFlagNotes](https://github.com/wmo-im/GRIB2/tree/codeFlagNotes)

[https://github.com/wmo-im/GRIB2/compare/dd7fa791680aab5ef6829671f61dffaf631d941a...codeFlagNotes](https://github.com/wmo-im/GRIB2/compare/dd7fa791680aab5ef6829671f61dffaf631d941a...codeFlagNotes)

**Comment by amilan17** [comment]
**For Review/Validation**

1. Code Table 3.2: 10
2. Code Table 4.5: 111 and 118
3. Code table 4.2: 2, 3, 4

**Comment by amilan17** [comment]
compare changes to notes

[https://github.com/wmo-im/GRIB2/compare/802f262357b3686ae95553a0ec8da8bc5e04d3bf..b4e344f9f65f5395484721bd9ab22bc117aac834](https://github.com/wmo-im/GRIB2/compare/802f262357b3686ae95553a0ec8da8bc5e04d3bf..b4e344f9f65f5395484721bd9ab22bc117aac834)

view notes: 

https://github.com/wmo-im/GRIB2/blob/codeFlagNotes/notes/codeTableNotes.csv

**Comment by amilan17** [comment]
@sebvi will you please take a look at this soon?

**Comment by sebvi** [comment]
I will try to have a look this evening

**Comment by sebvi** [comment]
I is very hard to follow what has changed from the compare because the number of changes is huge as you added the new column for the Id of the notes everywhere :(

Could have been easier to check if adding all the new columns were contained in 1 single commit so that I can filter it out. Is it the case?

**Comment by amilan17** [comment]
> is very hard to follow what has changed from the compare because the number of changes is huge as you added the new column for the Id of the notes everywhere :(

Right. I will improve this issue for clarity, by adding references and a useful diff. Please don't waste your time. I should have something by end of today or the week.

**Comment by amilan17** [comment]
# 1

\<img width="871" border="1" alt="Screenshot 2021-12-10 at 11 41 44" src="https://user-images.githubusercontent.com/5168866/145561393-4be7a238-755d-4d56-8e6e-72d1f536ebd2.png">

**Comment by amilan17** [comment]
![](https://user-images.githubusercontent.com/5168866/145562958-e54c83f8-4658-4a6a-9049-269e61aef6e5.png)

**Comment by amilan17** [comment]
**4.2.1.0**

**add existing notes to parameters** 

(1) Remotely sensed snow cover is expressed as a field of dimensionless, thematic values. The currently accepted values are for no-snow/no-cloud, 50, for clouds, 100, and for snow, 250 (see Code table 4.215).  
(2) A data field representing snow coverage by elevation portrays at which elevations there is a snow pack. The elevation values typically range from 0 to 90 in 100-metre increments. A value of 253 is used to represent a no-snow/no-cloud data point. A value of 254 is used to represent a data point at which snow elevation could not be estimated because of clouds obscuring the remote sensor (when using aircraft or satellite measurements).  
(3) Snow water equivalent per cent of normal is stored in per cent of normal units. For example, a value of 110 indicates 110 per cent of the normal snow water equivalent for a given depth of snow

![](https://user-images.githubusercontent.com/5168866/145571802-cf711de9-93da-489f-80c9-0ef404521e22.png)

**Comment by amilan17** [comment]
@sebvi -- I hope this provides more context -- apologies for the inconsistent presentations.

**Comment by amilan17** [comment]
all new notes look appropriate.
